test/dbtvault_harness_utils.py

```diff
@@ -129,6 +129,21 @@ def clean_csv(model_name=None):
             os.remove(file)
 
 
+def clean_sql(model_name=None):
```

Comment:
`clean_models` right below this function already does this.

Reply:
There are slight differences (the model name parameter) just re-factor `clean_models` rather than having two functions

Reply:
Yes, that's what I thought, but the existing function did not work for me, and this did.

Reply:
Can you re-factor it and see if it works please?

Reply:
I have refactored and committed changed code for this.